actions/st2_perform_migrations.sh

```diff
@@ -153,6 +153,13 @@ migrate_to_1.5() {
   run_migration_scripts $scripts
 }
 
+migrate_to_2.2() {
```

Comment:
Okay, roger that! So this must be something that is done regardless of version, then